tests/serialization/sparsity_pattern_empty.cc

```diff
@@ -0,0 +1,60 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2010 - 2018 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+
+// Check serialization for SparsityPattern. This used to fail for empty
+// sparsity patterns with a segfault.
+
+#include <deal.II/lac/sparsity_pattern.h>
+
+#include "../testmatrix.h"
+#include "serialization.h"
+
+
+void
+test()
+{
+  // Try a sparsity pattern that has nonzero size, but is otherwhise empty
```

Comment:
```suggestion
  // Try a sparsity pattern that has nonzero size, but is otherwise empty
```
(if you care to make this fix)